### PR TITLE
Add windows.graphics.holographics.interop

### PR DESCRIPTION
**Pull request by john-h-k**

### CODE REVIEW

**Comment by john-h-k** [comment]
@reflectronic github wont let me but i also request a review from u

File: samples/DirectX/Properties/launchSettings.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "TerraFX.Samples.DirectX": {
       "commandName": "Project",
-      "commandLineArgs": "all",
+      "commandLineArgs": "D3D12.HelloTriangle",
```
Comment (reflectronic) [inline samples/DirectX/Properties/launchSettings.json:5]: Intended for checkin?

File: samples/DirectX/Program.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@ public static unsafe class Program
             new HelloWindow11(1280, 720, "D3D11.HelloWindow"),
             new HelloTriangle11(1280, 720, "D3D11.HelloTriangle"),
             new HelloWindow12(1280, 720, "D3D12.HelloWindow"),
-            new HelloTriangle12(1280, 720, "D3D12.HelloTriangle"),
+            new HelloTriangle12(720, 720, "D3D12.HelloTriangle"),
```
Comment (reflectronic) [inline samples/DirectX/Program.cs:17]: Intended for checkin?

Comment (john-h-k) [reply samples/DirectX/Program.cs:17]: I thought I'd unstaged them. They were for debugging something else

File: generation/um/windows.graphics.holographic.interop/generate.rsp [hunk 3]
```diff
@@ -0,0 +1,15 @@
+@../../settings.rsp
```
Comment (tannergooding) [inline generation/um/windows.graphics.holographic.interop/generate.rsp:1]: Please add these files to the solution.

File: samples/DirectX/Properties/launchSettings.json [hunk 4]
```diff
@@ -1,9 +1,9 @@
 {
-  "profiles": {
-    "TerraFX.Samples.DirectX": {
-      "commandName": "Project",
-      "commandLineArgs": "all",
-      "nativeDebugging": true
+    "profiles": {
```
Comment (tannergooding) [inline samples/DirectX/Properties/launchSettings.json:2]: something is still changed here

**Comment by john-h-k** [comment]
Fixed the SLN stuff, all builds now @tannergooding

**Review by tannergooding — APPROVED** [review]

**Pull request merged**